integration-tests/tests/syscall_exit.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate contains integration tests, including the exit syscall payload test.
+
+#![deny(clippy::all)]
+#![deny(missing_docs)]
+
+mod common;
+use common::IntegrationTest;
+
+/// This test runs the exit syscall payload in the SGX keep using the SGX shim.
```

Comment:
Let's remove this comment. The TEE is chosen at compile-time.

Reply:
Fixed.